windows/rngallery/MainPage.cpp

```diff
@@ -17,19 +16,5 @@ namespace winrt::rngallery::implementation
         InitializeComponent();
         auto app = Application::Current().as<App>();
         ReactRootView().ReactNativeHost(app->Host());
-        // Workaround for https://github.com/microsoft/react-native-windows/issues/6287
```

Comment:
Revert? This looks like this should still be here.

Reply:
Sure. Note, I only ran the `react-init --override` command. No manual changes so far.